Cake.Recipe/Content/build.cake

```diff
@@ -132,10 +127,7 @@ BuildParameters.Tasks.DotNetCoreRestoreTask = Task("DotNetCore-Restore")
 {
     DotNetCoreRestore(BuildParameters.SolutionFilePath.FullPath, new DotNetCoreRestoreSettings
     {
-        Sources = new [] {
-            "https://api.nuget.org/v3/index.json",
-            "https://www.myget.org/F/cake-contrib/api/v3/index.json"
-        },
+        Sources = BuildParameters.NuGetSources,
```

Comment:
> AFAIK it does support V2 feeds

I see, in that case I don't see any problems at all.

> no V2 feeds specified anywhere in Cake Recipe?

Well there is though, you moved them to be the default of NuGetSources:
https://github.com/devlead/Cake.Recipe/blob/d32f599ce0efaf83d2cc5b92f2d05aeae5b5b7a8/Cake.Recipe/Content/parameters.cake#L363-L366